shared/login/register/select-other-device/index.render.desktop.js

```diff
@@ -5,12 +5,24 @@ import {globalStyles, globalColorsDZ2} from '../../../styles/style-guide'
 import Container from '../../forms/container.desktop'
 import type {Props} from './index.render'
 
-const Row = ({id, name, type, onSelect}) => {
-  const iconType = type === 'mobile' ? 'fa-mobile' : 'fa-laptop'
+const Row = ({deviceID, name, type, onSelect}) => {
+  const iconType = {
+    'mobile': 'fa-mobile',
+    'computer': 'fa-laptop',
+    'paper key': 'paper-key'
+  }[type]
+
+  const onClick = e => {
+    onSelect(deviceID)
+    e && e.preventDefault()
```

Comment:
Yes but we need this since in click changes the text right? Maybe that's
not true with dz2

On Friday, March 4, 2016, Marco Munizaga <notifications@github.com> wrote:

> In shared/login/register/select-other-device/index.render.desktop.js
> <https://github.com/keybase/client/pull/2228#discussion_r55112546>:
>
> > @@ -5,12 +5,24 @@ import {globalStyles, globalColorsDZ2} from '../../../styles/style-guide'
> >  import Container from '../../forms/container.desktop'
> >  import type {Props} from './index.render'
> >
> > -const Row = ({id, name, type, onSelect}) => {
> > -  const iconType = type === 'mobile' ? 'fa-mobile' : 'fa-laptop'
> > +const Row = ({deviceID, name, type, onSelect}) => {
> > +  const iconType = {
> > +    'mobile': 'fa-mobile',
> > +    'computer': 'fa-laptop',
> > +    'paper key': 'paper-key'
> > +  }[type]
> > +
> > +  const onClick = e => {
> > +    onSelect(deviceID)
> > +    e && e.preventDefault()
>
> because the onClick is on the text and the parent container
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/2228/files#r55112546>.
>